src/interfaces/chain.h

```diff
@@ -212,6 +212,7 @@ class Chain
     //! Return false if the transaction could not be added due to the fee or for another reason.
     virtual bool broadcastTransaction(const CTransactionRef& tx,
         const CAmount& max_tx_fee,
+        const CFeeRate& max_tx_feerate,
```

Comment:
in "[node]: update BroadcastTransaction to check fee rate limit" (https://github.com/bitcoin/bitcoin/pull/29278/commits/368d140db549387fb7bf162c96dd9e9064045103):

nit: in other places, you've been calling this `max_tx_fee_rate`.

Reply:
Fixed